markup.go

```diff
@@ -47,36 +47,32 @@ type Event struct {
 
 // MarkupOrComponentOrHTML represents one of:
 //
-//  Markup
+//  MarkupList
 //  Component
 //  *HTML
+//  ChildList
```

Comment:
`nil` should also be added here.

Reply:
Yeah, I noticed this after I left my other comments, but didn't want to force-push just for that.